### ISSUE TITLE
BO - Employee page - The alert is not well displayed when we cancel the modification of the password

### ISSUE THREAD

**Issue by khouloudbelguith** [PrestaShop/PrestaShop#29491]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

In the BO > Edit employee page, when we cancel the 'modification of the password' process, an alert is displayed and it looks bad and is not in the right position.

![image](https://user-images.githubusercontent.com/16067358/187659768-9df5d304-072c-4196-8f72-89c8e4b13cd3.png)

The alert pop-up is about the password being not secure enough. It does not make sense.

### Expected behavior

When we cancel the 'modification of the password' process, there should be no alert displayed. _We canceled the password change_ so no data was submitted no data was modified. There should be no alert.

### Steps to reproduce

1. Go to backoffice page BO > Advanced Parameters > Team > Employee page
2. Edit an employee
3. Click on 'change password' it opens a little div with the inputs needed to change password
4. Input inside "New password" with an not secure password, for example "123456789"
5. Click on 'Cancel'
6. See the problem with alert displayed in the wrong place 

![image](https://user-images.githubusercontent.com/16067358/187660351-dd5d6a15-8fe1-45f9-b520-2f153fb6e7bf.png)

**You can see a video below about the full step-by-step process.**

### PrestaShop version(s) where the bug happened

8.0.0 beta

### PHP version(s) where the bug happened

8.0

**Comment by AureRita** [comment]
Hi @khouloudbelguith ,

I reproduce the issue with PrestaShop version 8.0.0 Beta and not with the version 1.7.8.7. Nice catch ! It's a regression ! I'll add it to the patch backlog.
You can check the bug on this video : 

https://user-images.githubusercontent.com/100699445/187677128-fada2a35-72a7-435e-8d2f-8dfeb9e8b990.mp4


Please be aware that some issues might take a very long time to be resolved. 

Thank you

**Comment by margud** [comment]
I was able to reproduce this issue on 8.0.x and develop branch. Found that this issue is caused because of ps.form_widget_with_error in form.html.twig file line: 97.
When we cancel password change it is hidden but the error message is still displayed and moved to another position. Tried to find a fix but with no success. Looks like some modifications with macros.html.twig needs to be done. Maybe some one will find this information useful and will be able to fix it.
